modules/theme-tools/responsive-videos.php

```diff
@@ -11,9 +11,13 @@ function jetpack_responsive_videos_init() {
 
 	/* If the theme does support 'jetpack-responsive-videos', wrap the videos */
 	add_filter( 'wp_video_shortcode', 'jetpack_responsive_videos_embed_html' );
-	add_filter( 'embed_oembed_html',  'jetpack_responsive_videos_embed_html' );
 	add_filter( 'video_embed_html',   'jetpack_responsive_videos_embed_html' );
-
+	
+	/* Check to make sure the content is a video before applying to oEmbeds */
+	if ( ! empty( get_media_embedded_in_content( 'video' ) ) ) {
```

Comment:
`get_media_embedded_in_content ()` takes two parameters `$content, $type`.  We'll need to declare the first parameter `$content` before we give it a $type.  

Declaring an <a href="http://php.net/manual/en/function.empty.php#refsect1-function.empty-parameters">arbitrary expression within an empty() function</a> is only allowed >= PHP 5.5.